srcpkgs/kcat/template

```diff
@@ -1,17 +1,20 @@
-# Template file for 'kafkacat'
-pkgname=kafkacat
-version=1.6.0
+# Template file for 'kcat'
+pkgname=kcat
+version=1.7.0
 revision=1
 build_style=configure
 configure_args="--prefix=/usr"
-makedepends="librdkafka-devel"
+make_install_args="INSTALL=install"
+makedepends="librdkafka-devel yajl-devel"
 short_desc="Command line Apache Kafka producer and consumer"
 maintainer="Frank Steinborn <steinex@nognu.de>"
 license="BSD-2-Clause"
 homepage="https://github.com/edenhill/kafkacat"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=e67aecd153659d7ea0d561469a1e65038485fe9a3203fa8c9c59a9fe02164bf0
+checksum=baaed3cb6f5ade6a01355edb8819365e263a325375bee50727e009810e49cb38
+replaces="kafkacat>=1.6.0"
```

Comment:
The manual was missing a section on renaming packages, but assuming that @Chocimier opened https://github.com/void-linux/void-packages/pull/34875/files in response to this, I'd like to point you there. The way you renamed the package here is not sufficient.